packages/runtime-core/src/helpers/renderSlot.ts

```diff
@@ -38,7 +38,12 @@ export function renderSlot(
       currentRenderingInstance!.parent.isCE)
   ) {
     if (name !== 'default') props.name = name
-    return createVNode('slot', props, fallback && fallback())
+    return createVNode(
+      'slot',
+      props,
+      fallback && fallback(),
+      !!fallback ? PatchFlags.BAIL : undefined
```

Comment:
If there is a dynamic node in the backup slot, it will not enter the fast path anyway.🤔


https://deploy-preview-9213--vue-sfc-playground.netlify.app/#eNp9Uk1v2zAM/SucLmmBNj70FjjZR9DDdtiKbUddXJt21MqSIMlpAEP/fZTkj2DYerLJ98hHUm9kn43ZngdkO1a62grjwaEfzIEr0RttPYxgsYUArdU9bIi6WaCjVr4SCu0Ebouj7lM74nBVa+Wo3Um/wT42uWkr6fCWq7LIUiRCgcfeyMojRQDl8+C9VvCplqJ+3XM2lX+IX84SB8DrrpMI9zCOuX8IqbjI1bnTOt0uco4nIRuLar+J0WbqVDbiDOd70U5SiwRBzlTqMI4xHQLNHMOpqqCyrEIrTzIUl8WyDLtj3tEFWtFtX5xWdOAxFnBW042ERPvDeEEX4mwHCYlYJaV++5Zy3g54N+frE9av/8i/uEvMcfZk0aE9I2cL5ivboc/w46/veKH/Bex1M0hivwP+RKflEGfMtC+DamjsK16a9mvyglDdb/d48ajcvFQcNDJD4nNGtoj2+N/q67gP24dUx1WgK86WunKorFRH7+WpwexWSxUNtvQOT1YbV44pA8kd88t/3MGz1hIrlcBwuLmNP6sbKUiJ1ZGpSemkzijN6a8dk1w5t4/WyaJ/u2dmrC6aeJOPskqxyFzbKPwBp0U5PA==

Reply:
align https://github.com/vuejs/core/blob/b8fc18c0b23be9a77b05dc41ed452a87a0becf82/packages/runtime-core/src/helpers/renderSlot.ts#L75-L78

Reply:
> If there is a dynamic node in the fallback slot, it will not enter the fast path anyway.🤔

the fallback and slot nodes have different structures, so they cannot take the fast path.